src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -811,6 +827,22 @@
     <Resource Include="UI\Images\NodeStates\frozen-64px.png" />
     <Resource Include="UI\Images\NodeStates\hidden-64px.png" />
     <Resource Include="UI\Images\NodeStates\info-64px.png" />
+    <Resource Include="UI\Images\TitleBarButtons\close-darktheme-default-16px.png" />
```

Comment:
These are already added above. Duplicated.

Reply:
Can we go over those in chat, @reddyashish ?